include/picongpu/param/pml.param

```diff
@@ -33,11 +33,11 @@ namespace fields
 {
 namespace maxwellSolver
 {
-namespace yeePML
+namespace Pml
```

Comment:
Is this spelling what you agreed upon with @psychocoderHPC ? To be personally it's the least logical of all, but I am fine with any, just wanted to clarify

Reply:
Yes, because it is supposed to become a type (struct/class) soon anyway and than the naming does not need to be changed again.